File: w3/t2/person.js
class Person {
    /** @type {string} */
    first_name;
    last_name;
    /** @type {number} */
    #age;
    constructor(fname, lname, age) {
        this.first_name = fname;
        this.last_name = lname;
        this.#age = age;
    }
    getAge() {
        return this.#age;
    }
    ageUp() {
        this.#age++;
    }
    getFullname() {
        return this.first_name + ' ' + this.last_name;
    }
    printFullname() {
        console.log(this.getFullname());
    }
}

module.exports = Person;

File: w3/t1/person.py
class Person:
    first_name: str
    last_name: str
    __age: int
    def __init__(self, fname: str, lname: str, age: int) -> None:
        Person.first_name = fname
        Person.last_name = lname
        Person.__age = age
        return None
    def getAge(self) -> int:
        return Person.__age
    def ageUp(self) -> None:
        Person.__age += 1
        return None
    def getFullname(self) -> str:
        return f"{Person.first_name} {Person.last_name}"
    def printFullname(self) -> None:
        print(f"{Person.first_name} {Person.last_name}")
        return None